app/styles/partials/speaker.scss

```diff
@@ -0,0 +1,15 @@
+.speaker-item {
+  margin: 0;
+
+  .picture {
```

Comment:
The reason for using a custom tag is that the pictures do not take 100% width of its container. please check the screen shot and compare it with the previous version.

![screenshot from 2017-05-08 23-40-25](https://cloud.githubusercontent.com/assets/12807846/25818284/c0028af8-3447-11e7-9a08-ff91eb4e12d3.png)